include/mg_procedure.h

```diff
@@ -1405,36 +1405,53 @@ int mgp_must_abort(struct mgp_graph *graph);
 
 /// @}
 
-/// @name Kafka message API
-/// Currently the API below is for kafka only but in the future
-/// mgp_message and mgp_messages might be generic to support
-/// other streaming systems.
+/// @name Stream Source message API
+/// API for accessing specific data contained in a mgp_message
+/// used for defining transformation procedures.
+/// Not all methods are available for all streaming sources
+/// so make sure that your transformation procedure can be used
+/// for a specific source, i.e. only valid methods are used.
 ///@{
 
-/// A single Kafka message
+/// A single Stream source message
 struct mgp_message;
 
-/// A list of Kafka messages
+/// A list of Stream source messages
 struct mgp_messages;
 
 /// Payload is not null terminated and not a string but rather a byte array.
 /// You need to call mgp_message_payload_size() first, to read the size of
 /// the payload.
+/// Supoprted streaming sources:
```

Comment:
Okay we need to be consistent. `streaming sources` or `stream sources`? :thinking:

Reply:
Which one requires of you fewer changes ? Go with that

Reply:
It's the same actually because one of them is Python the other in C. :smile:

Reply:
Ohh I see, then I really don't have a strong preference :D Maybe go with `Stream sources`? @antaljanosbenjamin what do you think?